src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -215,12 +228,18 @@ def register_extra_cli_argument(command, dest, **kwargs):
     '''
     _cli_extra_argument_registry[command][dest] = CliCommandArgument(dest, **kwargs)
 
+def cli_command_with_handler(module_name, name, operation, client_factory=None, transform=None, table_transformer=None):
```

Comment:
The result isn't cached to file.
It's used so that when we come to load the params for a command, we know which module to load the params for.